Add release toolkit dictionary (NR-112268)

gsanchezgavier opened the pull request:
Adds a common dictionary used by the release toolkit on automatic pre-releases.

gsanchezgavier commented on rt-dictionary/.rt-dictionary.yml:
I will change default location and name of the file in the workflow that is used to run the rt

sigilioso reviewed (approved):
LGTM 👍 

Did you consider updating the README to mention this new file? (or the new configurable field)

gsanchezgavier commented:
> Did you consider updating the README to mention this new file? (or the new configurable field)
Thanks for pointing it @sigilioso ,I totally forgot. 
I modified and added some notes in the readme. LMKWDYT

sigilioso commented:
It looks great! Thanks for including the docs 🙂
